commands/publish_dafsa.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import tempfile
+from requests.exceptions import HTTPError
+
+import os
+import subprocess
+from kinto_http import Client, KintoException
+
+import json
+
+
+COMMIT_HASH_URL = (
+    "https://api.github.com/repos/publicsuffix/list/commits?path=public_suffix_list.dat"
+)
+LIST_URL = (
+    "https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat"
```

Comment:
`public_suffix_list.dat` could be a constant 

`COMMIT_HASH_URL = 'https:/.... ?path=%s' % FILENAME`